data.sql

```diff
@@ -1,6 +1,29 @@
 /* Populate database with sample data. */
+BEGIN;
 
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('1', 'Agumon', '2020-02-03', '0', 'true', '10.23');
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('2', 'Gabumon', '2018-11-15', '2', 'true', '8');
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('3', 'Pikachu', '2021-01-07', '1', 'false', '15.04');
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('4', 'Devimon', '2017-05-12', '5', 'true', '11');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03', '0', 'true', '10.23');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', '2', 'true', '8');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-07', '1', 'false', '15.04');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', '5', 'true', '11');
+
+SAVEPOINT day2;
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', '2020-02-08', '0', 'false', '11');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', '2021-11-15', '2', 'true', '5.7');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', '1993-04-02', '3', 'false', '12.13');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', '2005-06-12', '1', 'true', '45');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', '2005-06-07', '7', 'true', '20.4');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom', '1998-10-13', '3', 'true', '45');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', '2022-05-14', '4', 'true', '22');
```

Comment:
- [ ] Kindly make sure all the animals have the correct values on the `weight_kg` column as required for the projects especially the ones with negative numbers(e.g: **-5.7, -11.0, -45,...**)

Reply:
@tresorsawasawa Thanks for calling my attention to this, I have effected this request.